message_adapter/message_adapter.py

```diff
@@ -101,22 +102,39 @@ def loadRemoteEvent(self, event):
         )
         return self.loadAndUpdateRemoteEvent(event, None)
 
-    def loadAndUpdateRemoteEvent(self, event, context):
+    def __parseParameterConfiguration(self, event):
+        parsed_event = event
+        if event.get('cma'):
+            updated_event = {k:v for (k,v) in event['cma'].items() if k != 'event'}
+            parsed_event = event['cma']['event']
+            parsed_event.update(updated_event)
```

Comment:
So we pull the extra keys out of `event.cma` that aren't `event` and add them to the event returned by this method?

Reply:
Yes.  This is embarrassingly ugly because we can't use modern python conventions.   Thanks python 2! 

What this should look like is:
-            parsed_event = {**event['cma']['event'],
-                            **{k:v for (k,v) in event['cma'].items() if k != 'event'}}

Reply:
The effect is to wind up with the only information in the message being `event['cma']['event']` merged with event['cma'] (sans it's event key) at the top level of the returned event.  This has the effect of also removing all non cma parameters, which is desirable in this instance. 

This could probably be better refactored to read more clearly:
```
if event.get('cma'):
    updated_event = event['cma']
    del updated_event['cma']['event']
    return event['cma']['event'].update(updated_event) 
```

Reply:
OK. Your call on the refactor, just wanted to understand better